projects/demo/src/pages/documentation/maskito-libraries/maskito-libraries.template.html

```diff
@@ -4,121 +4,89 @@
         install them.
     </p>
 
-    <table class="tui-table">
-        <tbody>
-            <tr class="tui-table__tr">
-                <th class="tui-table__th">Library</th>
-                <th class="tui-table__th">About</th>
-            </tr>
-            <tr class="tui-table__tr">
-                <td class="tui-table__td">
-                    <a
-                        href="https://github.com/Tinkoff/maskito/tree/main/projects/core"
-                        target="_blank"
-                        tuiLink
-                    >
-                        <code>@maskito/core</code>
-                    </a>
-                </td>
-                <td class="tui-table__td">
-                    <p class="tui-space_top-0">
-                        It is the main zero-dependency and framework-agnostic
-                        package. It can be used alone in Vanilla Javascript
-                        project. It listens
-                        <code>beforeinput</code>
-                        and
-                        <code>input</code>
-                        events to validate and calibrate textfield's value.
-                    </p>
-                    <tui-doc-code
-                        code="npm install @maskito/core"
-                        filename="/your/project/path>"
-                    ></tui-doc-code>
+    <ul class="tui-list">
```

Comment:
In this file I've just replace table with list.

Table is not appropriate choice here (it is not readable with this content + bad for small screen devices).

Text content is the same.